torchtune/modules/tokenizers/_sentencepiece.py

```diff
@@ -38,16 +40,13 @@ def __init__(
         self.eos_id = spm_model.eos_id()
         self.pad_id = spm_model.pad_id()
 
-        # This is used in tokenize_messages: if the tokenizer does not
-        # encode whitespace, then we can more easily split strings
+        # If the tokenizer does not encode whitespace,
+        # then we can more easily split strings
         # on whitespace characters and encode them separately.
         self.encodes_whitespace = any(
             [self.spm_model.encode(c) for c in WHITESPACE_CHARS]
         )
```

Comment:
Shouldn't the test tokenizer already have `encodes_whitespace` properly defined here?

Reply:
Yes it is defined, it just turned out to be False